onnxruntime/test/providers/cpu/math/cumsum_test.cc

```diff
@@ -193,5 +193,19 @@ TEST(CumSumTest, _1DTestInt64) {
   test.AddOutput<int64_t>("y", {5}, {1, 3, 6, 10, 15});
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
 }
+TEST(CumSumTest, _1DTestdouble) {
```

Comment:
All the existing tests are quite exhaustive. Adding a couple more for completeness' sake.